unity-renderer/Assets/Scripts/MainScripts/DCL/AvatarSystem/Tests/WearableLoaderFactoryShould.cs

```diff
@@ -9,7 +12,14 @@ public class WearableLoaderFactoryShould
         private WearableLoaderFactory wearableLoaderFactory;
 
         [SetUp]
-        public void SetUp() { wearableLoaderFactory = new WearableLoaderFactory(); }
+        public void SetUp()
+        {
+            ServiceLocator serviceLocator = ServiceLocatorTestFactory.CreateMocked();
+            serviceLocator.Register<IWearablesCatalogService>(() => Substitute.For<IWearablesCatalogService>());
```

Comment:
cant you register `IWearablesCatalogService` into the `ServiceLocatorTestFactory` ?

Reply:
Yep, I can. Done: https://github.com/decentraland/unity-renderer/pull/4292/commits/a2a61c9b975b92d5ef662678f4a6bd2032e78f5a